modules/sno-du-cluster-specific-addressing.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-cluster-specific-addressing_{context}"]
+= Configuring cluster-specific addressing
+
+After networking has been configured, configure the specific addresses for the single node
+distributed unit (DU).
+
+.Procedure
+
+* The following example shows the configuration for IPv4 and IPv6 addresses. Configure IPv4 and IPv6
+on the network interface to support a dual stack deployment. Customize these
+settings for the specific environment of the single node DU:
++
+[source,yaml]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: NMStateConfig
+metadata:
+    labels:
+        nmstate-label: sno-du-1
+    name: sno-du-1
+    namespace: sno-du-1
+spec:
+    config:
+        dns-resolver:
+            config:
+                server:
+                    - 10.19.42.41 <1>
+        interfaces:
+            - ipv4:
+                Address: <2>
+                    - 10.16.231.3/24
+                    - 10.16.231.28/24
+                    - 10.16.231.31/24
+                dhcp: false
+                enabled: true
+              ipv6:
+                Address: <3>
+                    - 2620:52:0:10e7:e42:a1ff:fe8a:601/64
+                    - 2620:52:0:10e7:e42:a1ff:fe8a:602/64
+                    - 2620:52:0:10e7:e42:a1ff:fe8a:603/64
+                dhcp: false
+                enabled: true
```

Comment:
We have recently discontinued using three-address assignments for the SNO clusters due to deployment unreliability. It's now preferred to use a single IPv4/IPv6 address for all hostname roles (cluster, api.cluster, and *.apps.cluster), which can be accomplished using DNS CNAME entries.

Reply:
This topic has been removed.